doc/version_history.rst

```diff
@@ -6,6 +6,11 @@
 Version History
 ###############
 
+v.0.8.1
```

Comment:
There is an extra dot here between v and 0. 

Also, I think this is a minor release and not a patch so should be `v0.9.0`.